Library/ViewModels/LoginViewModel.swift

```diff
@@ -42,6 +42,9 @@ public protocol LoginViewModelInputs {
 
   /// Call when the view will appear.
   func viewWillAppear()
+  
+  /// Call when a trait collection is applied.
+  func traitCollectionApplied()
```

Comment:
We usually like to keep the input names as close as possible to the way it's named outside of the view model, so in this case it would be:

You can rename the input along with its backed properties, etc.

```swift
/// Call when a trait collection did change.
func traitCollectionDidChange()
```